ada/models.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
+
+
+class RedirectUrl(models.Model):
+
+    url = models.URLField(
+        max_length=255,
+        blank=True,
+        null=True,
+        default="https://hub.momconnect.za/confirmredirect",
+    )
```

Comment:
I'm not sure what this `url` field is being used for? It seems like it's only being used for display, but that seems odd. What is this URL?

Reply:
This doesn't seem to be resolved